### PR TITLE
Added DevToolsOptions.

### LINKED ISSUE #5008: [DevTools] Minimize the window
If you minimize the window on which the DevTools is attached, the DevTools window is also mininized.

### PR DESCRIPTION
**Pull request by grokys**
## What does the pull request do?

Tidied up #5011 and removed some options that didn't work/were unnecessary.

This PR adds a way to specify options for opening DevTools. Initial options are:

- Set the key gesture
- Show as child window
- Set the initial size

Co-Authored-By: @workgroupengineering

## Fixed issues

Fixes #5008

### CODE REVIEW

**Comment by grokys** [comment]
@workgroupengineering does this fit your needs?

**Comment by workgroupengineering** [comment]
Hi,

- On small screens LayoutVisualizer and StyleVisualizer making the datagrid with the properties unusable because the available space is small.
- I know they are already in the menu, but it's boring to set every FPS or DirtyRects , or whether or not to view the console.

**Comment by grokys** [comment]
> I know they are already in the menu, but it's boring to set every FPS or DirtyRects

You can already set them from code like this:

```csharp
        public MainWindow()
        {
            this.InitializeComponent();
            this.AttachDevTools();
            Renderer.DrawFps = true;
            Renderer.DrawDirtyRects = true;
```

> or whether or not to view the console.

Yes I can appreciate this but the code for initially showing the console didn't work anyway. Note that you can toggle the console with the Esc key as in Chrome etc.

File: src/Avalonia.Diagnostics/DevToolsExtensions.cs [hunk 1]
```diff
@@ -27,5 +27,15 @@ public static void AttachDevTools(this TopLevel root, KeyGesture gesture)
         {
             DevTools.Attach(root, gesture);
         }
+
+        /// <summary>
+        /// Attaches DevTools to a window, to be opened with the specified options.
+        /// </summary>
+        /// <param name="root">The window to attach DevTools to.</param>
+        /// <param name="options">additional settint of DevTools</param>
```
Comment (robloo) [inline src/Avalonia.Diagnostics/DevToolsExtensions.cs:35]: Spelling 'Settint' -> 'Settings' and probably should fix capitalization and period.

Comment (maxkatz6) [inline src/Avalonia.Diagnostics/DevToolsExtensions.cs:35]:
```suggestion
        /// <param name="options">Additional settings of DevTools.</param>
```

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**